package.json

```diff
@@ -5,5 +5,9 @@
   "main": "index.js",
   "repository": "https://github.com/codebase-berkeley/tgif-voting-portal.git",
   "author": "Warren <warrenwwang@berkeley.edu>",
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
+    "bootstrap": "^4.6.0",
+    "react-bootstrap": "^1.5.1"
+  }
```

Comment:
did u guys run `yarn add ___` in the root directory or in client directory? This seems like it should be in client/package.json